default.nix

```diff
@@ -9,6 +9,7 @@ in naersk.buildPackage {
     url = "https://github.com/fedimint/minimint";
     ref = "master";
   };
+  copyTarget = true; # copy the target directory to the build dir
```

Comment:
I assume that's to make use of the external rust caching?

Reply:
Yes